en/docs/concepts/authentication/intro-saml.md

```diff
@@ -0,0 +1,45 @@
+# Introduction to SAML
+
+## What is SAML?
+
+SAML stands for Security Assertion Markup Language, which is an XML based data format for exchanging authentication and authorization data between an identity provider and a service provider. The most important requirement that SAML addresses is web browser single sign-on (SSO). Three main roles are defined in SAML Specification.
+
+1. **The principal:** This is typically the user who tries to access a protected resource or service of a service provider.
+
+2. **The identity provider:** An Identity Provider (IdP) is responsible for authenticating users and issuing assertions which include authentication/authorization decisions and user attributes.
+
+3. **The service provider:** A Service Provider(SP) consumes the assertions issued by the identity provider and provides services to the principals.
+The main use case scenario covered by SAML is the principal (the user) requesting access to resource or service from the service provider. Then the service provider uses SAML to communicate with the identity provider and obtain an identity assertion. The service provider makes the access control decision, depending on this assertion.
+
+SAML 2.0 is the latest version of SAML, which uses security tokens containing assertions to pass information about a user between an identity provider and a service provider.
+
+---
+
+## The flow
+
+In a web browser-based SSO system, the flow can be triggered by the user, either by attempting to access a service at the service provider, or by directly accessing the identity provider itself.
+
+**If the user accesses a service at a service provider:**
```

Comment:
Shall we make this two points in a list or two subheadings? Making it bold seems a bit inconsistent. WDYT?